src/summary/cards/summary-card.component.tsx

```diff
@@ -19,7 +19,9 @@ export default function SummaryCard(props: SummaryCardProps) {
           )}
         </div>
         <div className={styles.headerAdd}>
-          <button className={`omrs-unstyled ${styles.addBtn}`}>Add</button>
+          {!props.hideHeaderRightButton && (
```

Comment:
I think we should do this in the opposite fashion, that is it should only be displayed if a route is provided such that adding will lead to that route